clients/include/testing_bsr2csr.hpp

```diff
@@ -280,8 +280,8 @@ hipsparseStatus_t testing_bsr2csr(Arguments argus)
     int                  m            = argus.M;
     int                  n            = argus.N;
     int                  block_dim    = argus.block_dim;
-    hipsparseIndexBase_t csr_idx_base = argus.idx_base;
-    hipsparseIndexBase_t bsr_idx_base = argus.idx_base2;
+    hipsparseIndexBase_t bsr_idx_base = argus.idx_base;
```

Comment:
Bug here as order of arguments passed in was not being accessed in the correct order in the testing_ code